src/blurredLocation.js

```diff
@@ -238,6 +238,7 @@ BlurredLocation = function BlurredLocation(options) {
     if(isBlurred()) {
         drawCenterRectangle(bounds);
         disableCenterMarker();
+        enableCenterShade() ; 
```

Comment:
Used this function here, similar to `disableCenterShade() ` function which is already used below .